shell/platform/android/io/flutter/embedding/android/FlutterView.java

```diff
@@ -164,9 +166,12 @@ public FlutterView(@NonNull Context context, @NonNull RenderMode renderMode) {
     if (renderMode == RenderMode.surface) {
       flutterSurfaceView = new FlutterSurfaceView(context);
       renderSurface = flutterSurfaceView;
-    } else {
+    } else if (renderMode == RenderMode.texture) {
       flutterTextureView = new FlutterTextureView(context);
       renderSurface = flutterTextureView;
+    } else {
+      throw new IllegalArgumentException(
+          String.format("RenderMode not supported with this constructor: ", renderMode));
```

Comment:
Hi, the missing format arguments break Google test (here and below line 268). If we have not yet, can we fix this?

Reply:
https://github.com/flutter/engine/pull/19261